app/src/main/java/io/pumpkinz/pumpkinreader/model/Comment.java

```diff
@@ -0,0 +1,31 @@
+package io.pumpkinz.pumpkinreader.model;
+
+
+import java.io.Serializable;
+import java.util.Date;
+
+public class Comment implements Serializable {
```

Comment:
I used Serializable to get it done quickly.
In Android, actually Parcelable is preferred.
I will, use Parcelable then.